tests/phpunit/tests/theme/wpGetGlobalStylesheet.php

```diff
@@ -246,6 +259,64 @@ public function test_caching_is_used_when_developing_theme() {
 		$this->assertNotSame( $css, wp_get_global_stylesheet(), 'Caching was used despite theme development mode' );
 	}
 
+	/**
+	 * Tests that theme color palette presets are output when appearance tools are enabled via theme support.
+	 *
+	 * @ticket 60134
+	 */
+	public function test_theme_color_palette_presets_output_when_border_support_enabled() {
+
+		$args = array(
+			array(
+				'name'  => 'Black',
+				'slug'  => 'nice-black',
+				'color' => '#000000',
+			),
+			array(
+				'name'  => 'Dark Gray',
+				'slug'  => 'dark-gray',
+				'color' => '#28303D',
+			),
+			array(
+				'name'  => 'Green',
+				'slug'  => 'haunted-green',
+				'color' => '#D1E4DD',
+			),
+			array(
+				'name'  => 'Blue',
+				'slug'  => 'soft-blue',
+				'color' => '#D1DFE4',
+			),
+			array(
+				'name'  => 'Purple',
+				'slug'  => 'cool-purple',
+				'color' => '#D1D1E4',
+			),
+		);
+
+		// Add theme support for appearance tools.
+		add_theme_support( 'border' );
+		add_theme_support( 'editor-color-palette', $args );
+		$this->remove_border_support_at_teardown = true;
```

Comment:
Similar to my other comment, it would be a lot simpler (no need for the member variable) if we just call `remove_theme_support()` at the end of the test.

Reply:
I was following the pattern of `$remove_theme_support_at_teardown` in this file. I agree simpler is better if there are no negative consequences to removing supports added per test.

Reply:
Would prefer to remove that as well if it's not necessary. Try it and if tests pass ✅